src/components/alerts/alert.jsx

```diff
@@ -23,19 +25,34 @@ const AlertComponent = ({
             ) : null}
             {message}
         </div>
-        <CloseButton
-            className={styles.alertCloseButton}
-            color={CloseButton.COLOR_ORANGE}
-            size={CloseButton.SIZE_LARGE}
-            onClick={onCloseAlert}
-        />
+        <button
+            className={styles.connectionButton}
+            onClick={onReconnect}
+        >
+            <FormattedMessage
+                defaultMessage="Reconnect"
+                description="Button to reconnect the device"
+                id="gui.connection.reconnect"
+            />
+        </button>
+        <Box
+            className={styles.alertCloseButtonContainer}
+        >
+            <CloseButton
+                className={styles.alertCloseButton}
+                color={CloseButton.COLOR_ORANGE}
+                size={CloseButton.SIZE_LARGE}
+                onClick={onCloseAlert}
+            />
+        </Box>
     </Box>
 );
 
 AlertComponent.propTypes = {
     iconURL: PropTypes.string,
     message: PropTypes.string,
-    onCloseAlert: PropTypes.func.isRequired
+    onCloseAlert: PropTypes.func.isRequired,
+    onReconnect: PropTypes.func.isRequired
```

Comment:
Similarly, I think we will want it possible to not supply `onReconnect` for alerts that don't have to do with extensions.

Reply:
👍